bower.json

```diff
@@ -7,8 +7,7 @@
     "ember-qunit-notifications": "0.1.0",
     "Faker": "~3.0.0",
     "pretender": "~0.10.1",
-    "lodash": "~3.7.0",
-    "bootstrap-sass": "^3.3.6"
+    "lodash": "~3.7.0"
```

Comment:
We were using the github app for simplicity (no ssh keys). Seems like it's "sync" feature may have tried to solve merge conflicts without user intervention.